packages/ra-auth-google/src/googleHttpClient.ts

```diff
@@ -0,0 +1,10 @@
+import { fetchUtils, Options } from "react-admin";
+
+export const googleHttpClient = async (url: string, options: Options = {}) => {
+  const token = localStorage.getItem("token");
```

Comment:
Can we have a security leak by storing Google token in the local storage?

Reply:
Maybe accept a Storage or a function as a 3rd parameter?

Reply:
Best method I know is https://marmelab.com/blog/2020/07/02/manage-your-jwt-react-admin-authentication-in-memory.html, but it requires having a server.
I can implement a function as a 3rd parameter though.